test_coverage/test_gcovr.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+# for info see 'read.me'
+
```

Comment:
add "set -e" and remove manual error handling
https://stackoverflow.com/questions/3474526/stop-on-first-error

Reply:
I agree, but when using "set -e" need to be careful:
http://mywiki.wooledge.org/BashFAQ/105